### DISCUSSION TITLE
How to get webview2 runtime earlier fixed version is 100.0.1185.29.

### DISCUSSION THREAD

**Discussion by Suresh-BabuLV** [MicrosoftEdge/WebView2Feedback#3975] [Q&A] [answered]
Hi Team,

We have created project to work on Webview2 runtime earlier version 100.0.1185.29.
Also tried from this website URL but not able to download earlier version
https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section

Please find the below image to see the 103 and 104 versions.
![image](https://user-images.githubusercontent.com/102970058/184175483-fd46564e-0368-4d41-b554-d2401d86414b.png)
 
Please guide and suggest to me to download the fixed version  100.0.1185.29

Great thanks in advance

Regards,
Suresh.

**Comment by westinyang** [comment, marked as answer]
@Suresh-BabuLV
I have archived WebView2 Fixed Version Runtime since the beginning of 2021, and currently I have collected versions 88 ~ 104. I created a repository and uploaded the archived files through Releases, hoping to help everyone!
https://github.com/westinyang/WebView2RuntimeArchive

**Comment by hyzhang** [comment]
downloadable exe: https://www.catalog.update.microsoft.com/Search.aspx?q=runtime%20edge%20118.0.2088.69
